README.md

```diff
@@ -68,6 +69,7 @@ Inspired by the [Developer Conferences](https://github.com/MurtzaM/Developer-Con
 | [Waterloo Codefest](http://www.waterloo.ca/en/government/WaterlooCodefest.asp) | Waterloo, Canada | 10.24 - 10.25 |
 | [LeadingAge HackFest](http://www.eventbrite.com/e/2015-leadingage-hackfest-hackathon-registration-11225376403?aff=es2) | Boston, MA | 10.30 - 11.1 |
 | [Boston Hacks](https://bostonhacks.io/) | Boston, MA | 10.31 - 11.1 |
+| [ecomHack Berlin](http://berlin.ecomhack.io/) | Berlin, Germany | 10.30 - 11.01 |
```

Comment:
Please list this in chronological order (then alphabetically), so between Waterloo Codefest and LeadingAge HackFest.